.github/workflows/build.yml

```
@@ -0,0 +1,80 @@
+name: Build

```

Reflect this change to the workflow file as well:

```suggestion
name: Build PR

```

---

This workflow can be manually dispatched as well. **Build** fits the name.